core/src/main/java/InitiateMatchStartCountdown/InitiateMatchStartCountdown.java

```diff
@@ -0,0 +1,31 @@
+package InitiateMatchStartCountdown;
```

Comment:
Per convention, no camel case in package names. Also, not too crazy about this name, too long imo

Reply:
I had to rebase the changes and while reimplenting them, Eclipse for some reason named the package that. I’ll make changes later today.